src/_guides/language/effective-dart/design.md

```diff
@@ -511,12 +528,14 @@ class Point {
 {% endprettify %}
 
 {:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/design_bad.dart (named-ctr)"?>
 {% prettify dart %}
 class Point {
   num x, y;
   Point(this.x, this.y);
   static Point polar(num theta, num radius) {
-    return new Point(radius * math.cos(theta),
+    return new Point(
+        radius * math.cos(theta),
```

Comment:
We should do what the formatter does, and it does:
```dart
class Point {
  num x, y;
  Point(this.x, this.y);
  static Point polar(num theta, num radius) {
    return new Point(radius * math.cos(theta), radius * math.sin(theta));
  }
}
```
or, preferably, use a `=>` function:
```dart
class Point {
  num x, y;
  Point(this.x, this.y);
  static Point polar(num theta, num radius) =>
      new Point(radius * math.cos(theta), radius * math.sin(theta));
}
```
If the line still gets too long, maybe remove the `math.` prefixes, which aren't necessary.

Reply:
Done. Dropped `math.` and switched to using `=>`.